lib/util.js

```diff
@@ -141,6 +147,11 @@ function styleText(format, text, { validateStream = true, stream = process.stdou
       right = `${escapeStyleCode(formatCodes[1])}${right}`;
     }
 
+    // Check colorize only after validating arg type and value
```

Comment:
Should we also move the check into the `for` loop? It's calling `escapeStyleCode` unnecessarily.

Reply:
The reason why I didnt move the return earlier is to also perform validation of the array values. While yes it is inefficient, if we returned early we would allow incorrect values.

Reply:
I didn't mean to skip validate, but skip escaping.

Reply:
the validation its in a for loop, so we either loop twice once to validate and once to format or we keep it as is. send a code change if you have a better a idea pls

Reply:
```js
    for (const key of format) {
      const formatCodes = inspect.colors[key];
      if (formatCodes == null) {
        validateOneOf(key, 'format', ObjectKeys(inspect.colors));
      }

      if (!skipColorize) {
        left += escapeStyleCode(formatCodes[0]);
        right = `${escapeStyleCode(formatCodes[1])}${right}`;
      }
    }
```